web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1244,50 +1244,102 @@ export const formatters: Formatters = {
   [eventCodes.DESKTOP_CLIPBOARD_RECEIVE]: {
     type: 'desktop.clipboard.receive',
     desc: 'Clipboard Data Received',
-    format: ({ user, desktop_addr, length }) =>
-      `User [${user}] received ${length} bytes of clipboard data from desktop [${desktop_addr}]`,
+    format: ({ user, desktop_addr, length, desktop_name }) => {
+      let desktopName = desktop_name ? desktop_name : desktop_addr;
+      let message = `User [${user}] received ${length} bytes of clipboard data from desktop [${desktopName}]`;
+      return message;
```

Comment:
```suggestion
      const desktop = desktop_name ?? desktop_addr;
      return `User [${user}] received ${length} bytes of clipboard data from desktop [${desktop}]`;
```

Reply:
That works too. Name the variable desktop though since it's not guaranteed to hold a name.